index.html

```diff
@@ -466,6 +470,9 @@ <h3 id="sec-processing-model"><span class="secno">4.4 </span>Processing Model</h
                 <p>Fire a simple event named <code>visibilitychange</code> that bubbles, isn't cancelable, and has no default action,
                 at the <a href="http://www.w3.org/TR/html5/dom.html#document">Document</a>.</p>
             </li>
+            <li>
+              <p>Run <a href="#document-is-visible">document is visible</a>.</p>

```

Comment:
What about something like:
Extension point: specifications that want to react when a <dfn>document became visible</dfn> can run their own algorithms here.

The <dfn> gives something to link to and it pushes specifications to not monkey patch but just use this as an event.
@marcoscaceres and @plehegar what do you think?